app/views/internal/configs/show.html.erb

```diff
@@ -105,6 +97,32 @@
           </div>
         </div>
 
+        <div class="card mt-3">
+          <%= render partial: "card_header",
+                     locals: {
+                       header: "Emails",
+                       state: "collapse",
+                       target: "emailBodyContainer",
+                       expanded: "false"
+                     } %>
+          <div id="emailBodyContainer" class="card-body collapse hide" aria-labelledby="emailBodyContainer">
+           <div class="form-group">
+             <%= f.fields_for :email_addresses do |email_field| %>
+               <% SiteConfig.email_addresses.keys.each do |email_type| %>
```

Comment:
I still insist on `SiteConfig.email_addresses.each do |email_type, email| ...end` instead of `keys.each` :D

Reply:
@lightalloy amended as per your suggestion